app/frontend/src/cljs/teet/admin/indexes_view.cljs

```diff
@@ -0,0 +1,343 @@
+(ns teet.admin.indexes-view
+  "Indexes admin user interface"
+  (:require [teet.ui.material-ui :refer [Table TableHead TableBody TableRow TableCell Paper Collapse]]
+            [teet.ui.text-field :refer [TextField]]
+            [teet.ui.buttons :as buttons]
+            [herb.core :refer [<class]]
+            [clojure.string :as str]
+            [teet.ui.icons :as icons]
+            [teet.localization :refer [tr tr-enum]]
+            [teet.ui.form :as form]
+            [teet.ui.format :as format]
+            [teet.ui.typography :as typography]
+            teet.user.user-spec
+            [teet.ui.select :as select]
+            [teet.ui.util :refer [mapc]]
+            [teet.ui.query :as query]
+            [reagent.core :as r]
+            [teet.theme.theme-colors :as theme-colors]
+            [teet.common.common-styles :as common-styles]
+            [teet.ui.common :as common-ui]
+            [teet.ui.context :as context]
+            [teet.user.user-model :as user-model]
+            [teet.util.date :as date]
+            [teet.ui.date-picker :as date-picker]
+            [teet.admin.admin-controller :as admin-controller]
+            [teet.ui.url :as url]
+            [cljs-time.core :as t]
+            [teet.util.coerce :as uc]))
+
+(defn- by-month [start-date]
+  (iterate #(t/plus % (t/months 1)) start-date))
+
+(def enabled-indexes
+  #{
+    {:name "Bitumen index"
+     :id "bitumen"
+     :type :cost-index.type/bitumen-index
+     :unit "€"}
+    {
+     :name "Consumer Price index"
```

Comment:
do the index names need translations?

Reply:
Index names are actually translated based on the :type - so the :name here is irrelevant.